sched/scheduler/task_runner.go

```diff
@@ -1,17 +1,34 @@
 package scheduler
 
 import (
+	"log"
+
 	"github.com/scootdev/scoot/common/stats"
 	"github.com/scootdev/scoot/saga"
 	"github.com/scootdev/scoot/sched"
 	"github.com/scootdev/scoot/sched/worker"
 	"github.com/scootdev/scoot/workerapi"
 )
 
+const DeadLetterExitCode = -100
```

Comment:
I think Unix exit codes are -128 to 127, so I might pick something like -200 just to be safely outside one byte?